lib/gherkin_lint/linter/tag_constraint.rb

```diff
@@ -0,0 +1,32 @@
+module GherkinLint
+  # Mixin to lint for tags that have certain string contraints
+  module TagConstraint
```

Comment:
We couldn't think of a way to do this.  That's why we set this linter to not be required, because we couldn't determine a sensible default.  So in the future, any additional implementations of the TagConstraint should be defaulted to disabled in the yml file.